src/components/Authentication/Logout.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react';
+import { instanceOf } from 'prop-types';
+import { logout, useNavigate } from '../../utils/auth_utils';
+import { Cookies, withCookies } from '../../utils/cookie_utils';
+
+const Logout = ({ cookies }) => {
+  const navigate = useNavigate();
+  const [errorMessage, setErrorMessage] = useState();
+
+  const handleLogout = async () => {
+    try {
+      await logout('/', navigate, cookies);
+    } catch (err) {
+      setErrorMessage(err.message);
+    }
+  };
+
+  return (
+    <div>
+      <h2>Logout</h2>
+      {errorMessage && <p>{errorMessage}</p>}
+      <button type="submit" onClick={handleLogout}>
+        Logout
+      </button>
+    </div>
+  );
+};
+
+Logout.propTypes = {
+  cookies: instanceOf(Cookies).isRequired,
+};
+
+export default withCookies(Logout);
```

Comment:
⚠️ **[eslint]** <react-refresh/only-export-components> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Fast refresh can't handle anonymous components. Add a name to your export.